ql/src/test/queries/clientpositive/udf_to_string.q

```diff
@@ -1,21 +1,34 @@
---! qt:dataset:src
+--! qt:dataset:alltypesorc
 set hive.fetch.task.conversion=more;
 
 -- Conversion of main primitive types to String type:
-SELECT CAST(NULL AS STRING) FROM src tablesample (1 rows);
+SELECT CAST(NULL AS STRING) FROM alltypesorc tablesample (1 rows);
 
-SELECT CAST(TRUE AS STRING) FROM src tablesample (1 rows);
+SELECT CAST(TRUE AS STRING) FROM alltypesorc tablesample (1 rows);
 
-SELECT CAST(CAST(1 AS TINYINT) AS STRING) FROM src tablesample (1 rows);
-SELECT CAST(CAST(-18 AS SMALLINT) AS STRING) FROM src tablesample (1 rows);
-SELECT CAST(-129 AS STRING) FROM src tablesample (1 rows);
-SELECT CAST(CAST(-1025 AS BIGINT) AS STRING) FROM src tablesample (1 rows);
+SELECT CAST(CAST(1 AS TINYINT) AS STRING) FROM alltypesorc tablesample (1 rows);
+SELECT CAST(CAST(-18 AS SMALLINT) AS STRING) FROM alltypesorc tablesample (1 rows);
+SELECT CAST(-129 AS STRING) FROM alltypesorc tablesample (1 rows);
+SELECT CAST(CAST(-1025 AS BIGINT) AS STRING) FROM alltypesorc tablesample (1 rows);
 
-SELECT CAST(CAST(-3.14 AS DOUBLE) AS STRING) FROM src tablesample (1 rows);
-SELECT CAST(CAST(-3.14 AS FLOAT) AS STRING) FROM src tablesample (1 rows);
-SELECT CAST(CAST(-3.14 AS DECIMAL(3,2)) AS STRING) FROM src tablesample (1 rows);
+SELECT CAST(CAST(-3.14 AS DOUBLE) AS STRING) FROM alltypesorc tablesample (1 rows);
+SELECT CAST(CAST(-3.14 AS FLOAT) AS STRING) FROM alltypesorc tablesample (1 rows);
+SELECT CAST(CAST(-3.14 AS DECIMAL(3,2)) AS STRING) FROM alltypesorc tablesample (1 rows);
 
-SELECT CAST('Foo' AS STRING) FROM src tablesample (1 rows);
+SELECT CAST('Foo' AS STRING) FROM alltypesorc tablesample (1 rows);
```

Comment:
Does the from clause relevant in these statements? It seem that we are casting constants.
Example:
```
SELECT CAST(CAST(-3.14 AS DOUBLE) AS STRING);
```
should also test `GenericUDFToString`. 

Could you please confirm?